OPHD/UI/NotificationArea.cpp

```diff
@@ -118,9 +118,23 @@ void NotificationArea::onMouseDown(EventHandler::MouseButton button, int x, int
 }
 
 
-void NotificationArea::onMouseMove(int x, int y, int dX, int dY)
+void NotificationArea::onMouseMove(int x, int y, int /*dX*/, int /*dY*/)
 {
+	if (!rect().contains({ x, y })) { return; }
 
+	size_t count = 0;
+	for (auto& rect : mNotificationRectList)
+	{
+		if (rect.contains({ x, y }))
+		{
+			mNotificationIndex = count;
+			return;
+		}
+
+		count++;
+	}
+
+	mNotificationIndex = SIZE_MAX;

```

Comment:
This would read better with a named constant that gave better semantics:
```cpp
mNotificationIndex = NoSelection;
```

Same for initialization:
```cpp
std::size_t mNotificationIndex{NoSelection};
```

For that, the class should probably define a `static constexpr` value that gives this a more meaningful name.
```cpp
static constexpr auto NoSelection{SIZE_MAX};
```

Or possibly even better, use a typed expression rather than the untyped `SIZE_MAX` pre-processor macro. That will ensure the `auto` deduces the correct type:
```cpp
static constexpr auto NoSelection{std::numeric_limits<std::size_t>::max()};
```

Reply:
I was thinking about doing something like that and then it just managed to poof out of my mind. I was looking for `std::numeric_limits<std::size_t>::max()` but... yeah. Go figure -_-